packages/apollo-link-http/package.json

```diff
@@ -42,7 +42,8 @@
     "watch": "tsc -w -p . & rollup -c -w"
   },
   "dependencies": {
-    "apollo-link": "^1.1.0"
+    "apollo-link": "^1.1.0",
+    "apollo-link-utilities": "0.0.0"
```

Comment:
The `apollo-link-utilities` dependency has been added, but it does not seem to be used yet in `apollo-link-http`?

Reply:
My mistake! The diff was hidden.